app/models/season/phase_switcher.rb

```diff
@@ -1,44 +1,60 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
+  PHASES = %i(
+    fake_proposals_phase
+    fake_application_phase
+    fake_coding_phase
+    back_to_reality
+    ).freeze
```

Comment:
Almost :v: (sorry for being so picky, but since it's so close...): put this line on the same level as `PHASES` and it looks adorable 😻